files/en-us/web/html/element/blink/index.html

```diff
@@ -26,7 +26,7 @@ <h2 id="Example">Example</h2>
 
 <h3 id="Result_toned_down!">Result (toned down!)</h3>
 
-<p><img alt="Image:HTMLBlinkElement.gif" src="/@api/deki/files/247/=HTMLBlinkElement.gif"></p>
+<p><img alt="Image:HTMLBlinkElement.gif" src="htmlblinkelement.gif"></p>
```

Comment:
Since the alt was just a form of the filename, it's better to just drop it till a proper one can be added
```suggestion
<p><img src="htmlblinkelement.gif"></p>
```

Reply:
Opened #937, so I can always rebase that if this one lands first